### PR TITLE
Removes Ethereal blood's shocking properties

### PR DESCRIPTION
**Pull request by ghost**
:cl: Qustinnus
balance: Ethereal blood no longer shocks.
balance: Liquid electricity found in food is now enriched, retaining the effect removed from the liquid electricity found in blood, which has less electrical properties. Enriched liquid electricity can also raise an ethereals blood levels
/:cl:

This splits up liquid electricity into 2 reagents:
non-enriched / default, found in Ethereals, does not shock or charge.

enriched: found in ethereal food, has the same properties as old ethereal blood except it also refills ethereal blood levels a bit. It's also a bit less strong than before.

### CODE REVIEW

**Comment by ATH1909** [comment]
:NOOOO:

File: code/modules/hydroponics/plant_genes.dm [hunk 1]
```diff
@@ -187,7 +187,7 @@
 
 /datum/plant_gene/reagent/liquidelectricity
 	name = "Liquid Electricity"
```
Comment (ATH1909) [inline code/modules/hydroponics/plant_genes.dm:189]:
```suggestion
	name = "Enriched Liquid Electricity"
```

File: code/modules/reagents/chemistry/reagents/food_reagents.dm [hunk 2]
```diff
@@ -808,7 +808,10 @@
 	taste_description = "pure electricity"
 	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
 
-/datum/reagent/consumable/liquidelectricity/expose_mob(mob/living/exposed_mob, methods=TOUCH, reac_volume) //can't be on life because of the way blood works.
+/datum/reagent/consumable/liquidelectricity/enriched
```
Comment (ATH1909) [inline code/modules/reagents/chemistry/reagents/food_reagents.dm:811]: it'd be nice if you added a chem recipe for ethereal blood enrichment

maybe something involving teslium?

Comment (Qustinnus) [reply code/modules/reagents/chemistry/reagents/food_reagents.dm:811]: maybe but not in this pr

File: code/modules/hydroponics/plant_genes.dm [hunk 3]
```diff
@@ -187,7 +187,7 @@
...
-	reagent_id = /datum/reagent/consumable/liquidelectricity
+	reagent_id = /datum/reagent/consumable/liquidelectricity/enriched
```
Comment (ATH1909) [inline code/modules/hydroponics/plant_genes.dm:190]: does enriched liquid electricity count as normal liquid electricity for reactions and such? if not, you might be removing some possible botany recipes here (including, say, electrolysis)

Comment (Qustinnus) [reply code/modules/hydroponics/plant_genes.dm:190]: not removed, just needs different reagents.

**Comment by theta006** [comment]
the blood should get enriched by being overcharged

**Comment by Qustinnus** [comment]
no.

**Comment by KathyRyals** [comment]
You forgot to explain why this is good for the game.

**Comment by Sparkezel** [comment]
> You forgot to explain why this is good for the game.

ever got shot by an ethereal?

**Comment by KathyRyals** [comment]
I'm just pointing out that he didn't fill the original post properly, I don't care about the nerf.

**Comment by EgorDinamit** [comment]
PR Template obviously states that if you can't explain why the addition is good for the game - it probably isn't worth it.
So...
image

**Comment by moocowswag** [comment]
If the shocks over time is really that annoying why not just condense all of the damage and knockdown to be (less in total) but instant and based on how much you recieved in a single injection? 

This would make ethereal blood something you need to spam large injections in order to keep someone (sort of) under control but still retains (and double downs) on its uniqueness as a weaponized chem.

**Comment by Qustinnus** [comment]
why this is good for the game is pretty self explanatory. It was not intended to be used like this in the first place

**Comment by moocowswag** [comment]
I seriously doubt someone would make ethereal blood hinder non ethereals and not intend for ethereals to be able to weaponize it.

**Comment by Qustinnus** [comment]
yet im telling you i did not intend for it.

**Review by Mothblocks — APPROVED** [review]

**Comment by Mothblocks** [comment]
i think you couldve tied it to purity, but it's fine since you can't actually make this anyway

**Pull request merged**